scripts/convert_csv_to_json.py

```diff
@@ -10,43 +10,47 @@ def generate_json_files():
+                if os.path.isfile(source_file_location):
 
-            if os.path.isfile(source_file_location):
+                    source_file = open(source_file_location, 'r')
 
-                source_file = open(source_file_location, 'r')
+                    output_file_name = source_file_name.replace('.csv', '.json')
+                    output_file_location = os.path.join(output_root,
+                                                        source_region_directory_name,
+                                                        language_directory_name,
+                                                        output_file_name)
 
-                output_file_name = source_file_name.replace('.csv', '.json')
-                output_file_location = os.path.join(output_root, source_directory_name, output_file_name)
+                    json_data = []
 
-                json_data = []
+                    with open(source_file_location, newline='') as csv_file:
+                        for row in csv.DictReader(csv_file, fieldnames=['term']):
+                            json_data.append(row.get('term'))
+                    try:
+                        with open(output_file_location, 'w', encoding='utf8') as f:
+                            json.dump(json_data, f, indent=3, ensure_ascii=False)
+                            f.write("\n")
+                    except FileNotFoundError:
+                        print(
+                            f"Output could not be written to {output_file_location}\nDoes the output folder"
+                            f"{output_root}/{source_region_directory_name}/{language_directory_name} exist?"
+                        )
+                        return
 
-                with open(source_file_location, newline='') as csv_file:
-                    for row in csv.DictReader(csv_file):
-                        # Append the data item as a json key-value if not an empty string
-                        json_data.append({k: v for k, v in row.items() if v})
-                try:
-                    with open(output_file_location, 'w', encoding='utf8') as f:
-                        json.dump(json_data, f, indent=3, ensure_ascii=False)
-                        f.write("\n")
-                except FileNotFoundError:
-                    print(
-                        f"Output could not be written to {output_file_location}\nDoes the output folder {output_root}/{source_directory_name} exist?"
-                    )
-                    return
+                    print(f"{source_file.name} >> {output_file_location}")
 
-                print(f"{source_file.name} >> {output_file_location}")
-
-                source_file.close()
-            else:
-                print(f"{source_file_location} is not a regular file")
+                    source_file.close()
+                else:
+                    print(f"{source_file_location} is not a regular file")
 
 
 if __name__ == "__main__":
```

Comment:
While we are here, can we sort out the spacing, it is tabbed one to many times.
```py
if __name__ == "__main__":
    generate_json_files()
```

Reply:
Don't blame me.. blame Pycharm's autoindent

Reply:
As I normally tell @JamesGardiner, #DodgySetup 😄